e2e/tests/visual/notification.visual.spec.js

```diff
@@ -0,0 +1,61 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2023, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/**
+ * This test is dedicated to test notification banner functionality and its accessibility attributes.
+ */
+
+const { test, expect } = require('../../pluginFixtures');
+const percySnapshot = require('@percy/playwright');
+
+test.describe('Visual - Check Notification Info Banner of \'Save successful\'', () => {
```

Comment:
I think this might fare better as a non-visual test, but we will need a better way to create notifications on the fly as part of an e2e test. Let me see if I can quickly implement some kind of appAction to generate a notification. You can then use that in your tests to focus on testing the new functionality you've added.

Reply:
@360macky: I've gone ahead and added an appAction for creating a notification within an e2e test. Please see `appActions.e2e.spec.js` for usage examples.

In the interest of getting this feature into our next release and not hanging it up on tests, I've split off adding the additional functional tests as a [separate issue](https://github.com/nasa/openmct/issues/6122) that can be completed in a separate PR.

Let's keep this visual test for now. I'm going to go ahead and review it, I'll probably have some suggestions for improvements there and once those are resolved we can merge this in!

Reply:
@unlikelyzero Check out / review the new appAction whenever you have some time. Should be fairly straightforward.

Reply:
> In the interest of getting this feature into our next release and not hanging it up on tests, I've split off adding the additional functional tests as a [separate issue](https://github.com/nasa/openmct/issues/6122) that can be completed in a separate PR.
> 
> Let's keep this visual test for now. I'm going to go ahead and review it, I'll probably have some suggestions for improvements there and once those are resolved we can merge this in!

Great! I checked the tests for `appActions.e2e.spec.js`, and they look good to me.